### ISSUE TITLE
Create an endpoint to save and retrieve data for Page Contents section

### ISSUE THREAD

**Issue by djhemath** [Dev-Profile-Page/dev-profile-page-backend#7]
There is a page in the app named "Page Contents". Need to create endpoints to save, update and retrieve the details.

**Comment by djhemath** [comment]
This depends on #8

**Comment by djhemath** [comment]
Completed by #11

**Comment by djhemath** [comment]
On submitting details for a page,

### New Record - if there is NO page ID in the request
1. Name, Bio, City and Website can be stored directly in `page` table.
2. For picture, we have to upload it to any storage services like GCP Cloud storage, AWS S3, etc., And then store the URL in `page` table.
3. Cards, required number of records in `page_card` should be created.
4. Technologies, required number of records in `page_technology` should be created.

### Update - if there IS page ID in the request
Only the updated fields should be sent to the server.

1. Name, bio, city and website - can be directly updated in `page` table
2. Picture - should be uploaded as a new picture, delete the old picture from storage and update the new URL in `page` table
3. Cards - can directly update whatever the cards are sent, if there is a new card, create new record
4. Technologies - check if new technology(ies) added, check if any technology(ies) deleted and perform necessary operations.
